### PR TITLE
Address CI failures and pretty-print bit enum operands

### PR DESCRIPTION
**Pull request by msiglreith**

### CODE REVIEW

**Comment by MarijnS95** [comment]
> and pretty-print bit enum operands

Otherwise it debug-prints the same enum variant double, right?

**Comment by msiglreith** [comment]
I would output `FunctionControl(INLINE | CONST)` for example in the test

File: rspirv/sr/storage.rs [hunk 1]
```diff
@@ -16,10 +16,7 @@ pub struct Token<T> {
 
 impl<T> Clone for Token<T> {
     fn clone(&self) -> Self {
-        Token {
-            index: self.index,
-            marker: self.marker,
-        }
+        *self
```
Comment (MarijnS95) [inline rspirv/sr/storage.rs:19]: Does this now call `Copy`... which calls `Clone`?

**Comment by MarijnS95** [comment]
Hmm for `Display` having just the variant name, but for `Debug` it seems nice to also include the type name at leaset once?

File: rspirv/sr/storage.rs (hunk 1 above)
Comment (msiglreith) [reply rspirv/sr/storage.rs:19]: `Copy` doesn't call `Clone` it's a memcpy internally, there is an example of this pattern in the rustdocs as well
https://doc.rust-lang.org/std/marker/trait.Copy.html#how-can-i-implement-copy

**Comment by msiglreith** [comment]
Switched back to  the default `Debug` impl and use `Display` instead to hide the duplicate type names, needed to touch some parts of the autogenerator

File: rspirv/dr/constructs.rs [hunk 2]
```diff
@@ -392,9 +392,11 @@ mod tests {
         assert_eq!(
             format!(
                 "{}",
-                dr::Operand::FunctionControl(spirv::FunctionControl::INLINE)
+                dr::Operand::FunctionControl(
+                    spirv::FunctionControl::INLINE | spirv::FunctionControl::CONST
+                )
             ),
-            "INLINE",
+            "INLINE | CONST",
```
Comment (MarijnS95) [inline rspirv/dr/constructs.rs:399]: Here for example, isn't it useful to know that we're printing an _`Operand` of variant `FunctionControl`_?

Comment (msiglreith) [reply rspirv/dr/constructs.rs]: Ok, I adjust the testcase to add the struct name and using the default `Debug` impl. (Initially I thought this testcase was for pretty-printing in disassembly but this seems to be separate)

Comment (MarijnS95) [reply rspirv/dr/constructs.rs]: Lovely, thanks so much!

**Review by MarijnS95 — APPROVED** [review]

**Pull request merged**